process/css-styleguide/button/button.css

```diff
@@ -0,0 +1,108 @@
+@import "../default.css";
+
+.btn {
+  border-radius: 2rem;
+  border-width: 0.1rem;
+  border-style: solid;
+  padding: 0.8rem 1.5rem;
+  cursor: pointer;
+  outline: none;
+  transition: 0.3s ease;
+  transition-property: background-color, border-color;
+  color: var(--white);
+}
+
+.btn-dodger-blue {
+  background-color: var(--dodger-blue-500);
+  border-color: var(--dodger-blue-500);
+}
+
+.btn-hot-pink {
+  background-color: var(--hot-pink-500);
+  border-color: var(--hot-pink-500);
+}
+
+.btn-lila {
+  background-color: var(--lila-500);
+  border-color: var(--lila-500);
+}
+
+.btn-turquoise {
+  background-color: var(--turquoise-600);
+  border-color: var(--turquoise-600);
+}
+
+.btn-watermelon {
+  background-color: var(--watermelon-500);
+  border-color: var(--watermelon-500);
+}
+
+.btn-peppermint {
+  background-color: var(--peppermint-500);
+  border-color: var(--peppermint-500);
+}
+
+.btn-candlelight {
+  background-color: var(--candlelight-600);
+  border-color: var(--candlelight-600);
+}
+
+.btn-disable {
+  background-color: var(--ashes-900);
+  border-color: var(--ashes-900);
+  pointer-events: none;
+}
+
+.btn-secondary {

```

Comment:
I don't think this class suits all, primary and secondary buttons use different colors for the different states (default, hover, and disabled). In the example, when you hover on the secondary button, the color applied is the color for the primary one
![Capture d’écran 2022-07-28 à 15 00 16](https://user-images.githubusercontent.com/48251481/181511246-10ca4bf0-5d2e-4618-a7d4-6e2f4f4eb766.png)

It should be
![Capture d’écran 2022-07-28 à 15 00 55](https://user-images.githubusercontent.com/48251481/181511318-79b22d37-4aec-4929-a2d8-84f193e9b8d3.png)